core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -170,7 +106,7 @@ class HttpSamDAO(baseSamServiceURL: String, serviceAccountCreds: Credential)(imp
                                    ctx: RawlsRequestContext
   ): Future[SamPolicySyncStatus] =
     retry(when401or5xx) { () =>
-      val callback = new SamApiCallback[SyncStatus]("syncStatus")
+      val callback = new SamApiCallback[sam.model.SyncStatus]("syncStatus")
```

Comment:
I consistently added the package prefix to all model classes coming from the sam client lib to try to keep things straight